public/apps/configuration/panels/permission-list/permission-list.tsx

```diff
@@ -301,7 +330,11 @@ export function PermissionList(props: AppDependencies) {
   };
 
   const createActionGroupMenuItems = [
-    <EuiButtonEmpty key="create-from-blank" onClick={() => showEditModal('', Action.create, [])}>
+    <EuiButtonEmpty
+      key="create-from-blank"
+      id="create-from-blank"
```

Comment:
we don't need `id` here as we are not using it in tests anymore

Reply:
Removed